lib/taskana-core-test/pom.xml

```diff
@@ -58,6 +58,12 @@
       <artifactId>postgresql</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.tngtech.archunit</groupId>
+      <artifactId>archunit</artifactId>
+      <version>${version.archunit}</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
please remove this dependency from the `taskana-core` module

Reply:
Ok :)